src/3d/spread3d_wrapper.cu

```diff
@@ -20,7 +20,18 @@ int CalcGlobalIdx(int xidx, int yidx, int zidx, int onx, int ony, int onz,
 	oix = xidx/bnx;
 	oiy = yidx/bny;
 	oiz = zidx/bnz;
-	return   (oix+oiy*onx + oiz*onx*ony)*(bnx*bny*bnz) +
+#if 0
```

Comment:
Do you have a DEBUG sort of flag you are using elsewhere? Some people dislike leaving debug prints around, but generally I don't mind if it is not disruptive to program because it helps understand what the developer was thinking or struggling with.  

However, if we leave it, its nice that it is pretty standard through out a code, and maybe gets tested once and a while, which again, is easier if there is a flag(s) to flip.

Do you think this is something that is useful to keep around for the next developer (which might be you :) )?

Reply:
Hi Garrett, sorry for the late reply. Yes, the debug codes should at least be standard through out the the code. I'll remove statements like `#if 0` in another pull request. 

I think for now it would be nice to keep the codes inside `#ifdef DEBUG`. They are helpful for me to see what went wrong through out the computation and might also helpful for other potential developers. But if you have better idea about how to manage the debug statements, please let me know, thanks!